bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +42,50 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_end():
+            line_nr_end = False
+            enumerated_zip_obj = zip(range(1, len(file) + 1),
+                                     file)
+            enumerated_tuple = tuple(enumerated_zip_obj)
+
+            for line_number, line in enumerated_tuple:
+                replacement = line
```

Comment:
There is no need for a temporary variable. Python uses **Call by Object Reference** and not **Pass by reference**.